### PR TITLE
Chat UI Fixes

### PR DESCRIPTION
**Pull request by edaena**
## Purpose

* Remove redundant buttons from the Chat Header
* Integrate the Chat under the sidebar
Screen Shot 2021-01-25 at 2 09 32 PM

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by navzam — APPROVED** [review]
I noticed that when I send a long chat message, for some participants, it makes the chat area expand a lot into the video area. Not sure if this is related to the header changes or if it was a problem before.

**Review by bnookala — APPROVED** [review]

**Comment by edaena** [comment]
@navzam just updated the design, please see image above for details.

File: Calling/ClientApp/src/components/Header.tsx [hunk 1]
```diff
@@ -47,6 +47,12 @@ export default (props: HeaderProps): JSX.Element => {
       : setSelectedPane(CommandPanelTypes.None);
   };
 
+  const toggleChat = (selectedPane: string, setSelectedPane: (pane: string) => void) => {
+    return selectedPane !== CommandPanelTypes.Chat
+      ? setSelectedPane(CommandPanelTypes.Chat)
+      : setSelectedPane(CommandPanelTypes.Chat);
```
Comment (navzam) [inline Calling/ClientApp/src/components/Header.tsx]: Should the second expression be `CommandPanelTypes.None`?

File: README.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ Additional documentation for this sample can be found on [Microsoft Docs](https:
 
 ![Homepage](./Media/homepage-sample-calling.png)
 
+![CallScreen](./Media/call-screen.png)
```
Comment (navzam) [inline README.md:15]: Not seeing this image in `./Media/`

**Review by navzam — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (navzam) [reply README.md:15]: Nevermind :)

**Pull request merged**